submissions/natashafir/js-dom/style.css

```diff
@@ -0,0 +1,116 @@
+body {
+    margin: 0;
+    padding: 0;
+    background-color: #162F3E;
+    color: white;
+    font-family: 'Raleway', sans-serif;
+}
+
+h1 {
+    margin: 0;
+    padding: 0;
+}
+
+img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.title {
+    display: flex;
+    justify-content: center;
+    margin-top: 20px;
+}
+
+.wrapper {
+    display: flex;
+    justify-content: center;
+}
+
+.content {
+    display: flex;
+    margin-top: 16px;
+    justify-content: space-around;
+    position: relative;
+    padding: 0 20px;
+    width: 900px;
+    height: 648px;
+}
+
+.navigation {
+    width: 30%;
+    flex: 1;
+}
+
+#nav__wrapper {
+    display: flex;
+    flex-direction: column;
+
+}
+
+.main {
+    padding: 10px;
+    background-color: #fff;
+    width: 56%;
+}
+
+.btn {
+    height: 3em;
+    font-size: 18px;
+    color: #73828b;
+    background-color: #132a37;
+    border: 1px solid #112531;
+}
+
+.btn:hover {
```

Comment:
Add `cursor: pointer;` here. It will be better for the user experience.

Reply:
hi, i stuck with git, not with task, I did a lot of mistakes with git and now can't push my changes

Reply:
maybe I can help you to solve your problems with Git? You can describe your issue here or in questionarium chat.